src/renamer/Renamer.cpp

```diff
@@ -71,6 +71,15 @@ QString Renamer::replaceCondition(QString& text, const QString& condition, bool
     return text;
 }
 
+bool Renamer::replaceDelimiter(QString& text, QString oldDelimiter, QString newDelimiter, bool execCondition)
```

Comment:
The third argument seems strange. I think it would be better to remove it and move the boolean into a condition surrounding the call to `replaceDelimiter`:

```cpp
if (execCondition) {
  Renamer::replaceDelimiter(...)
}

Reason being: It is not clear to the caller what the third argument does.

Also: `sanitizeFolderName` already calls `sanitizeFileName` in your PR. Why call `sanitizeFileName` after `replaceDelimiter` in `*Renamer.cpp`? :)